tg_bot/modules/sql/locks_sql.py

```diff
@@ -14,6 +15,7 @@ class Permissions(BASE):
     voice = Column(Boolean, default=False)
     contact = Column(Boolean, default=False)
     video = Column(Boolean, default=False)
+    videonote = Column(Boolean, default=False)
```

Comment:
What is the sql command to add this new column, for the migration to complete successfully? Sqlalchemy does not automatically migrate.

Reply:
ups! You are right, can you give me an idea of how to do it?